src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const married = isMarried
```

Comment:
```suggestion
  const marriedStatus = isMarried
```
Better to specify that it's string and not boolean, because "married" sound like a boolean value